### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by devmalik7**
Hey, hope you doing good.
I have optimized this README file.I have added the links of Catrobat Home website moreover i have added help link for the beginner user who is now familiar with the contribution at CATROBAT.Hope you like it.

*Please enter a short description of your pull request and add a reference to the Jira ticket.*

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s gitflow workflow
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Catroid #
 
-**Catroid** is a visual coding IDE and interpreter for Android for the Catrobat programming language.
+**Catroid** is a visual coding IDE and interpreter for Android Development for the Catrobat programming language.
 
-**Catrobat** is a visual programming language and a set of creativity tools for smartphones. 
+**Catrobat** is a visual programming language and a set of creativity tools for smartphones Android apps available on Google Play and iPhone apps available on Apple's app store.
```
Comment (devmalik7) [inline README.md:5]: to make it clear for user.

File: README.md [hunk 2]
```diff
@@ -25,6 +32,7 @@ Also make sure to read our guidelines for [creating a pull request](https://gith
 
 
 # Resources and links #
+* [Home Catrobat](https://catrobat.org/)
```
Comment (devmalik7) [inline README.md:35]: It is the link of Home page of Catrobat websites

File: catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java [hunk 3]
```diff
@@ -3,17 +3,15 @@
...
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
- *
  * An additional term exception under section 7 of the GNU Affero
  * General Public License, version 3, is available at
  * http://developer.catrobat.org/license_additional_term
- *
  * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * But WITHOUT ANY WARRANTY; without even the implied warranty of
```
Comment (wslany) [inline catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java:14]: Sorry, the AGP copyleft header text must remain _unchanged_, see https://www.gnu.org/licenses/gpl-howto.html

File: automationScripts/createBetaTestingPocketCodeAPK.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# does all of this for you:
+# This is done all for you:
```
Comment (wslany) [inline automationScripts/createBetaTestingPocketCodeAPK.py:1]: I think (not sure though) that this has to remain as it was.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,9 @@
 # Catroid #
 
-**Catroid** is a visual coding IDE and interpreter for Android for the Catrobat programming language.
+**Catroid** is a visual coding IDE and interpreter for Android Development for the Catrobat programming language.
```
Comment (wslany) [inline README.md:3]:
```suggestion
**Catroid** is a visual coding IDE and interpreter for Android for the Catrobat programming language.
```

File: README.md (hunk 1 above)
Comment (wslany) [inline README.md:5]:
```suggestion
**Catrobat** is a visual programming language and a set of creativity tools for smartphones.
```

**Review by wslany — CHANGES_REQUESTED** [review]
Thanks! A few changes please.

**Pull request closed without merging**